packages/eslint-plugin/tests/rules/no-confusing-non-null-assertion.test.ts

```diff
@@ -0,0 +1,159 @@
+import rule from '../../src/rules/no-confusing-non-null-assertion';
```

Comment:
Instead of disabling the lint rule for each test, just do it for the entire file ([example](https://github.com/typescript-eslint/typescript-eslint/blob/fe4c0bf8c04f070d6642fbe86c5e5614bc88e8fd/packages/eslint-plugin/tests/rules/brace-style.test.ts))

```suggestion
/* eslint-disable eslint-comments/no-use */
// this rule enforces adding parens, which prettier will want to fix and break the tests
/* eslint "@typescript-eslint/internal/plugin-test-formatting": ["error", { formatWithPrettier: false }] */
/* eslint-enable eslint-comments/no-use */

import rule from '../../src/rules/no-confusing-non-null-assertion';
```